webapp/utils/utils.jsx

```diff
@@ -314,14 +313,9 @@ export function getTimestamp() {
 }
 
 // extracts links not styled by Markdown
-export function extractLinks(text) {
-    text; // eslint-disable-line no-unused-expressions
-    Autolinker; // eslint-disable-line no-unused-expressions
-
-    // skip this operation because autolinker is having issues
-    return [];
-
-    /*const links = [];
+export function extractFirstLink(text) {
+    const pattern = /(^|[\s\n]|<br\/?>)((?:https?|ftp):\/\/[\-A-Z0-9+\u0026\u2019@#\/%?=()~_|!:,.;]*[\-A-Z0-9+\u0026@#\/%=~()_|])/gi;
```

Comment:
I think it's probably okay since it's just used for pulling out YouTube and image links so it won't cause too many problems if it incorrectly matches something. In the long term, I'd still like to modify the text formatting code to just output the links that it matched so that this extra processing isn't necessary, but this should work for now.